src/Stripe.net/Services/_base/Service.cs

```diff
@@ -290,11 +300,6 @@ protected RequestOptions SetupRequestOptions(RequestOptions requestOptions)
                 requestOptions = new RequestOptions();
             }
 
-            if (string.IsNullOrEmpty(requestOptions.ApiKey) && !string.IsNullOrEmpty(this.ApiKey))
```

Comment:
Can you clarify why this disappeared? Is it because the StripeClient itself will do the validation?

Reply:
So previously, there were 4 (!) different ways to set up an API key. In order of decreasing priority:

1. `RequestOptions.ApiKey`
2. `Service.ApiKey`
3. `StripeClient.ApiKey`
4. `StripeConfiguration.ApiKey`

The removed lines implemented #2. Basically "if `RequestOptions.ApiKey` is not already set (i.e. the user did not set an API key via #1) and `Service.ApiKey` is set, then set `RequestOptions.ApiKey` to `Service.ApiKey` (so that down the road the client will use this as the API key for the request).

After this PR, #2 is gone.